RT.jenkinsfile

```
@@ -86,9 +86,9 @@ pipeline {
 			}
 
 			post {
-				success {
-					dir('./norma_data_a1_latest') {
-						archiveArtifacts artifacts: '**'
+				always {
+					script {
+						uploadArtifacts(["*.yml", "*.csv", "*.log", "*.html"])
 					}
 				}
```

```suggestion
				always {
					uploadArtifacts(["*.yml", "*.csv", "*.log", "*.html"])
				}
```
There's no need to wrap it it `script{}` block